packages/odyssey-storybook/package.json

```diff
@@ -20,7 +20,9 @@
     "@mui/material": "^5.15.10",
     "@okta/odyssey-design-tokens": "workspace:*",
     "@okta/odyssey-react-mui": "workspace:*",
-    "@storybook/theming": "^7.2.0"
+    "@storybook/theming": "^7.2.0",
+    "react": "^18.2.0",
```

Comment:
@bryancunningham-okta In order to use `memo` I had to move React from `devDependencies` to `Dependencies` in order to import it since this is a system page in Storybook, rather than just a `.story` file in our mui package. Not sure if there are any concerns with this.